patterns/embedding/qdrant.py

```diff
@@ -0,0 +1,123 @@
+from typing import List
+
+from qdrant_client import QdrantClient
+from qdrant_client.models import Distance, PointIdsList, PointStruct, VectorParams
+
+from resolver import pattern
+
+from ..secret import Secret
+from .vectordb import VectorDB
+
+
+@pattern(name="Qdrant")
+class Qdrant(VectorDB):
+    """
+    VectorDB implementation using Qdrant.
+    """
+
+    def __init__(self, url: str, api_key: Secret = None):
```

Comment:
Can't do this because [resolve_params](https://github.com/pingcap/LinguFlow/blob/main/api/application.py#L56) method can't handle compose type at present.

Reply:
Got it.